doc/release/1.17.0-notes.rst

```diff
@@ -33,6 +33,16 @@ The internal use of these functions has been refactored and there are better
 alternatives. Relace ``exec_command`` with `subprocess.Popen` and
 ``temp_file_name`` with `tempfile.mkstemp`.
 
+Writeable flag of C-API wrapped arrays
+--------------------------------------
+When an array is created from the C-API to wrap already existing data and
+the array does not own its data (frees it after destruction) it will not be
+possible to switch its writeable flag to ``True`` in the future.
```

Comment:
Maybe clearer to state the reason:

"When an array is created from the C-API to wrap a pointer to data, the only indication we have of the read-write nature of the data is the ``writeable`` flag set during creation. It is dangerous to force the flag to ``writeable``.  In the future it will not be possible to switch the writeable flag to ``True``.

Reply:
Yeah, that is better, added a "from python" at the end.